relayer/src/foreign_client.rs

```diff
@@ -554,6 +555,66 @@ impl<DstChain: ChainHandle, SrcChain: ChainHandle> ForeignClient<DstChain, SrcCh
         self.build_update_client_with_trusted(target_height, Height::zero())
     }
 
+    fn get_trusted_height_from_client_state(
```

Comment:
Minor suggestion, feel free to ignore. we typically drop `get` from func names in `ibc-rs`. Also it is not always the one from client state. Maybe something like:
```suggestion
    fn solve_trusted_height(
```

Reply:
d6c5f78